cmd/thanos/store.go

```diff
@@ -106,104 +110,106 @@ func runStore(
 	chunkPoolSizeBytes uint64,
 	maxSampleCount uint64,
 	maxConcurrent int,
-	component string,
+	component component.Component,
 	verbose bool,
 	syncInterval time.Duration,
 	blockSyncConcurrency int,
 	filterConf *store.FilterConfig,
 ) error {
-	{
-		confContentYaml, err := objStoreConfig.Content()
-		if err != nil {
-			return err
-		}
+	statusProber := prober.NewProber(component, logger, prometheus.WrapRegistererWithPrefix("thanos_", reg))
```

Comment:
Could we not wrap metrics with `thanos_` prefix in this PR, we could have it in another one. We have alerts and dashboards depend on this, better to do this change explicitly.

Reply:
Hm are we changing anything? Before we did not have any probe metrics so I think this should be fine?

Reply:
Hmm as Bartek pointed out this does not change anything just adds new metrics. Is that an issue for you?

Reply:
lgtm